docker/rosbridge/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM ros:melodic-ros-core
+
+SHELL ["/bin/bash", "-c"]
+RUN apt-get update && apt-get install -y ros-melodic-rosbridge-suite \
+    && useradd --create-home --shell /bin/bash spaceuser \
+    && mkdir -p /home/spaceuser/Programming/robotics-prototype
+
+
+WORKDIR /home/spaceuser/Programming/robotics-prototype
+COPY requirements.txt requirements-dev.txt setup.py docker/rosbridge/entrypoint.sh ./
+
+RUN apt-get update && apt-get install -y python3-pip \
+    && pip3 install -r requirements.txt -r requirements-dev.txt \
+    && pip3 install rosdep \
+    && python3 setup.py develop \
+    && rm -rf /var/lib/apt/lists/* 
+
```

Comment:
Adding "RUN apt-get update -y && apt-get upgrade -y" here fixed my errors with building the docker images.

Reply:
You might as well commit it my man or make a suggest it through GitHub

Reply:
Yea I guess I'll just commit real quick then